app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -57,7 +53,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     private val COARSE_LOCATION = Manifest.permission.ACCESS_COARSE_LOCATION
     private lateinit var marker: Marker
     lateinit var mvm: MainViewModel
-    private var GEOFENCE_RADIUS = 500F
+    private val GEOFENCE_RADIUS = 500F
```

Comment:
The naming conventions of this variable are for a constant, so I changed it assuming it is supposed to be a constant. https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
Good.